src/addons/editors/DropDownEditor.js

```diff
@@ -32,8 +32,12 @@ class DropDownEditor extends EditorBase {
 
   renderOptions(): Array<ReactElement>{
     var options = [];
-    this.props.options.forEach(function(name){
-      options.push(<option key={name} value={name}  >{name}</option>);
+    this.props.options.forEach(function(name, value){
+      if typeof value === 'string' && typeof value === 'number'
```

Comment:
think we'd prefer a undefined check here, otherwise something that looks like an object (ie `typof new Date() === 'object'` would not set the value)